.gitignore

```diff
@@ -1 +1,2 @@
-/node_modules
\ No newline at end of file
+node_modules
+src/Database/Config/dbConfig.ts
```

Comment:
dbConfig.ts should not be added to .gitignore. it is important for the project to execute correctly